README.md

```diff
@@ -92,9 +92,13 @@ flavors:
 deployment:
   # kind, Supported values are terraform, pulumi, crossplane.
   kind: terraform
-  # Context for deployment: the context including some kind of parameters for the deployment, such as fix,variable.
-  # - env: The value of the fix parameters are defined by the ISV with the @value at the initial time.
-  # - variable: The value of the variable parameters are defined by the user on the console.
+  # Context for deployment: the context including some kind of parameters for the deployment, such as fix_env, fix_variable, env, variable, env_env, env_variable.
```

Comment:
Based on our discussion, please see if we can replace the description as below.

- FIX_ENV:  Values for variable of this type are defined by the managed service provider in the OCL template. Runtime will inject it to deployer as enviroment variables. This varible is not visible to the end user. 

- FIX_VAR: Values for variable of this type are defined by the managed service provider in the OCL template. Runtime will inject it to deployer as usual variables. This varible is not visible to the end user. 

- ENV: Value for a variable of this type can be provided by end user. If marked as mandatory then end user must provide value to this variable. If marked as optional and if end user does not provided it,  then the fallback value to this variable is read by runtime (it can read from other sources, e.g., OS env variables). This variable is injected as a  environment variable to the deployer. 

- VAR: Value for a variable of this type can be provided by end user. . If marked as mandatory then end user must provide value to this variable. If marked as optional and if end user does not provided it, then the fallback value to this variable is read by runtime (it can read from other sources, e.g., OS env variables). This variable is injected as a  regular variable to the deployer.

- ENV_ENV:  Value to this variable is read by runtime (it can read from other sources, e.g., OS env variables) and injected as a  environment variable to the deployer. End user cannot see or change this variable.

- ENV_VARIABLE: Value to this variable is read by runtime (it can read from other sources, e.g., OS env variables) and injected as a  regular variable to the deployer. End user cannot see or change this variable.